spec/index.bs

```diff
@@ -2573,7 +2573,8 @@ interface GPUAdapter {
 
                     <div class=validusage>
                         1. |adapter|.{{adapter/[[state]]}} must not be {{adapter/[[state]]/"consumed"}}.
-                        1. For each [|key|, |value|] in |descriptor|.{{GPUDeviceDescriptor/requiredLimits}}:
+                        1. For each [|key|, |value|] in |descriptor|.{{GPUDeviceDescriptor/requiredLimits}}
+                            for which |value| is not `undefined`:
                             1. |key| |must| be the name of a member of [=supported limits=].
```

Comment:
Kelsey suggested a non-normative note to suggest a warning when an unknown key is specified with `undefined`, I think this is a good idea, will add it before landing.